datadogreceiver/README.md

```diff
@@ -0,0 +1,93 @@
+# Datadog APM Receiver
+
+## Overview
+The Datadog APM Receiver accepts traces in the Datadog Trace Agent Format.
+
+This module is Experimental. The code was originally written in a closed PR from the opentelemetry-collector-contrib repo and then updated to the latest standards. The intent is to contribute this back into the opentelemetry-collector-contrib.
```

Comment:
Please can you add link to the closed PR to give context back to why it stalled in contrib first time.